drivers/gemaltomodem/voicecall.c

```diff
@@ -133,7 +133,11 @@ static void gemalto_hangup_all(struct ofono_voicecall *vc,
 static void gemalto_hangup(struct ofono_voicecall *vc,
 				ofono_voicecall_cb_t cb, void *data)
 {
-	unsigned int affected = (1 << CALL_STATUS_ACTIVE);
+	unsigned int affected = (1 << CALL_STATUS_ACTIVE) |
+			(1 << CALL_STATUS_DIALING) |
+			(1 << CALL_STATUS_ALERTING) |
+			(1 << CALL_STATUS_INCOMING);
+	DBG("Hanging up ACTIVE");
```

Comment:
please remove the DBG here (also because it could not be 'ACTIVE' but 'ALERTING' or other).
AT^SHUP will be used later on to terminate a specific call, instead of the current AT+CHUP that terminates all.
AT+CHLD is unsuitable here because it can only terminate active calls, according to the 3GPP 27.007